shells/dev-shell/index.js

```diff
@@ -43,6 +45,7 @@ const {
 init();
 
 function init() {
+  VolatileStorage.setStorageCache(new RuntimeCacheService());
```

Comment:
Just to note, if something else uses Runtime.getRuntime() in the dev-shell control flow (even indirectly, via storage or storageNG code), it will end up accessing a different `RuntimeCacheService` instance.

I'd like to remove all Runtime.getRuntime() calls but have not yet.

If this works for now we're probably ok but wanted to raise awareness.

Reply:
The only testing that I did was to load a Kotlin particle in a dev shell and make sure the exception goes away.